First commit! (#5)
WayScience/IC_Test: README.md

# Microscopy Imaging Illumination and Contrast Evaluation

> ![MIICE logo](./media/logo/MIICE_logo_v1.png)

Reviewer: This won't show in the preview properly because the repo hasn't yet been renamed and the work isn't checked into `main` but once those things happen this image will be ready for display on external systems like PyPI (which otherwise won't show the image as-is).
```suggestion
![MIICE logo](https://raw.githubusercontent.com/wayscience/MIICE/main/media/logo/MIICE_logo_v1.png?raw=true)
```

Author: Ahh okay so this is getting ahead of the game, okay cool I like it! I will change it now!

> > Don’t let your images squeak by untested! 🐭🧀

Reviewer: This is just grate!

Author: HAHA! Yes! I love the mouse puns this is going to be awesome lol

MIICE is a Python package to evaluate microscopy images for quality issues like uneven illumination and poor contrast prior to perform image analysis.

It is nearly impossible to acquire "perfect" images across large high-throughput screens.
Technical artifacts from the microscope can impact all of the images (like uneven illumination) or some of the images (out-of-focus, over-saturation, or poor contrast).

> Most modern microscopes have the ability to try and correct for this, but it is not always full proof.

Reviewer: ```suggestion
Most modern microscopes can attempt to correct image quality issues, but these corrections are not foolproof and they rarely are available to analysts.
```

Author: Added!

> Most modern microscopes have the ability to try and correct for this, but it is not always full proof.

Reviewer 2: Small suggestion: It might be more effective to introduce the problem first and then present MIICE as the solution. This paragraph is strong and engaging, so placing it at the beginning could help draw the reader in right away.

Author: This is a great suggestion! I will implement that now! Thanks Erik!

> With MIICE, you can test your microscopy images directly from the microscope (raw) or after illumination correction is performed to test the quality of the illumination and contrast.

Reviewer: ```suggestion
With MIICE, you can test unmodified or corrected microscopy images to test the quality of the illumination and contrast.
```

Author: Much more concise!

The results can be used to determine next steps, including:

1. Decision to perform illumination correction on a microscopy dataset that was said to be already corrected by the microscope.
2. Confirm optimal parameters or refine for illumination correction on a dataset.
3. Flag and/or filter out low contrast image sets in a dataset.

## Installation

> TBD

Reviewer: Consider adding in installation from source:

````suggestion
```shell
pip install "git+https://https://github.com/WayScience/MIICE"
```
````

Author: Ahh duh, thanks for that suggestion I forgot about source!
